src/test/java/uk/gov/hmcts/darts/audio/component/impl/AddAudioRequestMapperImplTest.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.hmcts.darts.audio.component.impl;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.darts.audio.model.AddAudioMetaDataRequest;
+import uk.gov.hmcts.darts.common.entity.CourthouseEntity;
+import uk.gov.hmcts.darts.common.entity.CourtroomEntity;
+import uk.gov.hmcts.darts.common.entity.MediaEntity;
+import uk.gov.hmcts.darts.common.repository.CourtroomRepository;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.Optional;
+
+import static org.mockito.Mockito.anyString;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class AddAudioRequestMapperImplTest {
+    @Mock
+    CourtroomRepository courtroomRepository;
+
+    AddAudioRequestMapperImpl addAudioRequestMapperImpl;
+
+    @BeforeEach
+    void setUp() {
+        addAudioRequestMapperImpl = new AddAudioRequestMapperImpl(courtroomRepository);
+    }
+
+    @Test
+    void testMapToMedia() {
+        CourthouseEntity courthouse = new CourthouseEntity();
+        courthouse.setCourthouseName("SWANSEA");
+
+        CourtroomEntity courtroomEntity = new CourtroomEntity(1, "1", courthouse);
+        when(courtroomRepository.findByCourthouseNameAndCourtroomName(anyString(), anyString())).thenReturn(Optional.of(
+            courtroomEntity));
+
+        OffsetDateTime start = OffsetDateTime.now().minusHours(1);
+        OffsetDateTime endDate = OffsetDateTime.now();
+
+        MediaEntity media = new MediaEntity();
+        media.setStart(start);
+        media.setEnd(endDate);
+        media.setChannel(1);
+        media.setTotalChannels(2);
+        media.setCourtroom(courtroomEntity);
+        media.setCaseIdList(List.of("case1,case2"));
```

Comment:
It probably doesn't matter much - but did you want to split these 2 caseIds?

Reply:
Not sure what you mean by this. do you mean add a space in-between?